services/hanke-service/src/main/resources/db/changelog/changesets/065-add-attachment-sizes.sql

```diff
@@ -0,0 +1,23 @@
+--liquibase formatted sql
+--changeset Teemu Hiltunen:065-add-attachment-sizes
+--comment: Add column size to application_attachment and hanke_attachment tables
+
+-- Application attachment size
+ALTER TABLE application_attachment
+    ADD COLUMN size BIGINT default 0;
+
+COMMENT ON COLUMN application_attachment.size IS 'Size of the attachment in bytes';
+
+UPDATE application_attachment
+SET size = length(content)
+FROM application_attachment_content
+WHERE application_attachment.id = application_attachment_content.attachment_id;
+
+
+-- Hanke attachment size
+ALTER TABLE hanke_attachment
+    ADD COLUMN size BIGINT default 0;
```

Comment:
The default values could be dropped after the existing data has been initialized.

Reply:
Should these be NOT NULL?

Reply:
`content_type` column seems to have a default value `default 'application/octet-stream'::text` - should that be removed as well? I now dropped default for size only.

I added NOT NULL for `size`

Reply:
Dropping the default value from `content_type` columns wouldn't hurt. Not mandatory, though.